### DISCUSSION TITLE
InternalError: too much recursion

### DISCUSSION THREAD

**Discussion by ilia-ilia14** [ismail9k/vue3-carousel#224] [Q&A] [answered]
I am using this component on my vue3 app. it works perfectly fine on my local dev environment. but as soon as I deploy it starts to give me  "InternalError: too much recursion" errors and component totally stops working. is this a bug or I am implementing it wrong ? here are the screenshots of the error.
 https://drive.google.com/file/d/1JFU9-8fM-WZ1kOCIzkrFMN4JeEs24PaP/view?usp=sharing 
https://drive.google.com/file/d/1-9jx3puyfZokpHE_Y03DgaGz5jqo18F8/view?usp=sharing
![too_much_errors1](https://user-images.githubusercontent.com/22649411/190837757-21e11d90-0304-4ab3-aa03-bb86b485561e.JPG)
![too_much_errors](https://user-images.githubusercontent.com/22649411/190837760-28eca596-863f-4ad0-9bd4-9e6e187334c8.JPG)

**Comment by ismail9k** [comment, marked as answer]
This issue should be fixed in the latest version
